build/crd-samples/router/create-rule-rest-eventbus.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rules.kubeedge.io/v1
+kind: Rule
+metadata:
+  name: my-rule
+  labels:
+    description: test
+spec:
+  source: "my-rest"
+  sourceResource: {"path":"/a"}
```

Comment:
Use a more meaningful path even it is only an example.

Reply:
how about modifying it to "/test"

Reply:
OK.

Reply:
Miss?

Reply:
ok